packages/roosterjs-editor-api/lib/format/clearFormat.ts

```diff
@@ -10,13 +12,61 @@ import { ChangeSource, DocumentCommand, IEditor, QueryScope } from 'roosterjs-ed
 
 const STYLES_TO_REMOVE = ['font', 'text-decoration', 'color', 'background'];
 
+export const enum FormattingStrategy {

```

Comment:
Move this type decaration into roosterjs-editor-types package, and use a better name to show it is for clear format only

Reply:
Alright, I'll use ClearFormatMode